src/test/java/com/fullteaching/e2e/no_elastest/common/BaseLoggedTest.java

```diff
@@ -66,7 +66,7 @@ static void setupAll() { // 28 lines
             // Load a properties file for reading
             properties.load(new FileInputStream("src/test/resources/inputs/test.properties"));
         } catch (IOException ex) {
-            ex.printStackTrace(); // Consider logging the exception instead of printing the stack trace
+            log.error("Properties file could not be loaded");
```

Comment:
As I understand, the test.properties file is a requirement to run the tests. If it can't be loaded, the test should fail